block-producer/src/txqueue/mod.rs

```diff
@@ -158,16 +183,26 @@ where
     TV: TransactionVerifier,
     BB: BatchBuilder,
 {
+    #[allow(clippy::blocks_in_conditions)] // Workaround of `instrument` issue
+    #[instrument(target = "miden-block-producer", skip_all, err)]
     async fn add_transaction(
         &self,
         tx: SharedProvenTx,
     ) -> Result<(), AddTransactionError> {
+        info!(target: COMPONENT, tx_id = %tx.id().inner(), account_id = %tx.account_id());
```

Comment:
Similar comment to the one before: we should now be able to use `to_hex()`for both account ID and transaction ID.